packages/circuit-ui/components/ButtonGroup/ButtonGroup.tsx

```diff
@@ -55,7 +56,7 @@ const wrapperStyles = ({ theme }: StyleProps) => css`
   width: 100%;
 
   ${theme.mq.kilo} {
-    flex-direction: row;
+    flex-direction: row-reverse;
```

Comment:
We reverse the button order on larger viewports. This means that the tab order will be consistent on all viewports (first stop is the primary action, second stop is the secondary action)